src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -117,8 +117,17 @@ EMSCRIPTEN_BINDINGS(ClpStreamReader) {
     // JS types used as inputs
     emscripten::register_type<clp_ffi_js::ir::DataArrayTsType>("Uint8Array");
     emscripten::register_type<clp_ffi_js::ir::LogLevelFilterTsType>("number[] | null");
+    emscripten::register_type<clp_ffi_js::ir::ReaderOptions>(
+            "{logLevelKey: string, timestampKey: string} | null"
+    );
 
     // JS types used as outputs
+    emscripten::enum_<clp::ffi::ir_stream::IRProtocolErrorCode>("IRProtocolErrorCode")
```

Comment:
If we end up exposing IRProtocolErrorCode, let's name it `IrProtocolErrorCode`. I think it's better to avoid propagating the naming errors from CLP.